mlflow/server/js/src/components/ParallelCoordinatesPlotPanel.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ParallelCoordinatesPlotView from './ParallelCoordinatesPlotView';
+import { ParallelCoordinatesPlotControls } from './ParallelCoordinatesPlotControls';
+import { getSharedMetricKeysByRunUuids, getSharedParamKeysByRunUuids } from '../reducers/Reducers';
+import _ from 'lodash';
+
+import './ParallelCoordinatesPlotPanel.css';
+
+export class ParallelCoordinatesPlotPanel extends React.Component {
+  static propTypes = {
+    runUuids: PropTypes.arrayOf(String).isRequired,
+    sharedParamKeys: PropTypes.arrayOf(String).isRequired,
+    sharedMetricKeys: PropTypes.arrayOf(String).isRequired,
+  };
+
+  static defaultProps = {
+    sharedParamKeys: [],
+    sharedMetricKeys: [],
+  };
+
+  state = {
+    selectedParamKeys: this.props.sharedParamKeys, // default select all parameters
+    selectedMetricKeys: this.props.sharedMetricKeys.slice(0, 1),
+  };
+
+  handleParamsSelectChange = (paramValues) => {
+    this.setState({ selectedParamKeys: paramValues });
+  };
+
+  handleMetricsSelectChange = (metricValues) => {
+    this.setState({ selectedMetricKeys: metricValues });
+  };
+
+  render() {
+    const { runUuids, sharedParamKeys, sharedMetricKeys } = this.props;
+    const { selectedParamKeys, selectedMetricKeys } = this.state;
+    return (
+      <div className='parallel-coorinates-plot-panel'>
+        <ParallelCoordinatesPlotControls
+          paramKeys={sharedParamKeys}
+          metricKeys={sharedMetricKeys}
+          selectedParamKeys={selectedParamKeys}
+          selectedMetricKeys={selectedMetricKeys}
+          handleMetricsSelectChange={this.handleMetricsSelectChange}
+          handleParamsSelectChange={this.handleParamsSelectChange}
+        />
+        {(!_.isEmpty(selectedParamKeys) || !_.isEmpty(selectedMetricKeys)) ? (
+          <ParallelCoordinatesPlotView
+            runUuids={runUuids}
+            paramKeys={selectedParamKeys}
+            metricKeys={selectedMetricKeys}
+          />
+        ) : null}
```

Comment:
nit: The UX here is a bit weird since the height of the window will jump back and forth between `null` and `ParallelCoordinatesPlotView`, when I toggle the last metric/param on and off.

Reply:
Good point. CC @gioa , what do you think about this, we probably can show a placeholder with some message when there is nothing to plot.